docs/commands/bundle.md

```diff
@@ -1,117 +1,91 @@
-```
+:::success Tip
+To learn more about preprocessors, rules, and decorators, please refer to [this](../custom-rules.md) document.
+:::
 
-The command:
+## Usage
 
 ```bash
-openapi bundle --output <outputName> --ext <ext> [entrypoints...]
-```
-
-- `[entrypoints...]` corresponds to the name(s) of your root document(s).
-- Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
-- The `<outputName>` is your desired output filename or folder. If the folder doesn't exist, it's automatically created.
-- Use `--ext <ext>` to specify the type and extension of the output file: `.json`, `.yml` or `.yaml`.
+openapi bundle <entrypoints>...
+openapi bundle <entrypoints> [--max-problems=<n>]
+openapi bundle <entrypoints> [--lint] [--config=<path>]
+openapi bundle <entrypoints>... -o <outputName> --ext <ext>
+openapi bundle --version
+```
+
+## Options
+
+option                 | type      | required?    | default     | description
+-----------------------|:---------:|:------------:|:-----------:|------------
+`entrypoints`          | `array`   | yes          | `[]`        | Array of API root definition filenames that need to be bundled. Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
+`--config`             | `string`  | no           | -           | Specify path to the config file
+`--dereferenced`, `-d` | `boolean` | no           | -           | Generate fully dereferenced bundle
+`--ext`                | `string`  | no           | `yaml`      | Specify bundled file extension.<br />**Possible values:** `json`, `yaml`, `yml`
+`--force`, `-f`        | `boolean` | no           | -           | Generate bundle output even when errors occur
+`format`               | `string`  | no           | `codeframe` | Format for the output.<br />**Possible values:** `codeframe`, `stylish`
+`--help`               | `boolean` | no           | -           | Show help
+`--lint`               | `boolean` | no           | `false`     | Lint definition files
+`--max-problems`       | `number`  | no           | 100         | Truncate output to display the specified maximum number of problems
+`--output`, `-o`       | `string`  | no           | -           | Name or folder for the bundle file. For example, `-o bundle.yaml` or `-o ./openapi`.<li>If you don't specify the extension, `.yaml` will be used by default.</li><li>If the specified folder doesn't exist, it will be created automatically.</li><br />**If the file specified as the bundler's output already exists, it will be overwritten**
+`--skip-decorator`     | `array`   | no           | -           | Ignore certain decorators
+`--skip-preprocessor`  | `array`   | no           | -           | Ignore certain preprocessors
+`--skip-rule`          | `array`   | no           | -           | Ignore certain rules
```

Comment:
question: Can the user specify which decorators/preprocessors/rules will be ignored? If yes, could we show an example how to do that?

Reply:
Please check the content starting from the line [#77](https://github.com/Redocly/openapi-cli/pull/350/files#diff-00c97ff1398f245dc4a49f3ea92c5827a829e8332501f762fd1720ea05069402R77). If believe this is a short, yet descriptive example.

Reply:
Circling back on this - yeah, that example does exactly what I was looking for :+1: Could you add a sentence to the table with a link to that section? Similar to how we have those "see section below" links in tables for other commands.

Reply:
Added links to the section